itests/src/test/java/fi/metatavu/edelphi/test/mock/QueryMocker.java

```diff
@@ -0,0 +1,272 @@
+  }
+
+  private void deleteQueryPage(Long pageId) {
+    executeSql("DELETE FROM QueryQuestionOptionGroupOptionAnswer where id in (SELECT id FROM QueryQuestionOptionAnswer WHERE option_id in (SELECT id FROM QueryOptionFieldOption WHERE optionField_id in (SELECT id from QueryField WHERE queryPage_id = ?)))", pageId);
+    executeSql("DELETE FROM QueryQuestionOptionAnswer WHERE option_id in (SELECT id FROM QueryOptionFieldOption WHERE optionField_id in (SELECT id from QueryField WHERE queryPage_id = ?))", pageId);
+    executeSql("DELETE FROM QueryQuestionTextAnswer WHERE id in (SELECT id FROM QueryQuestionAnswer WHERE queryField_id in (SELECT id from QueryField WHERE queryPage_id = ?))", pageId);
+    executeSql("DELETE FROM QueryQuestionNumericAnswer WHERE id in (SELECT id FROM QueryQuestionAnswer WHERE queryField_id in (SELECT id from QueryField WHERE queryPage_id = ?))", pageId);
+    executeSql("DELETE FROM __QueryQuestionMultiOptionAnswers WHERE answer_id in (SELECT id FROM QueryQuestionAnswer WHERE queryField_id in (SELECT id from QueryField WHERE queryPage_id = ?))", pageId);
+    executeSql("DELETE FROM QueryQuestionMultiOptionAnswer WHERE id in (SELECT id FROM QueryQuestionAnswer WHERE queryField_id in (SELECT id from QueryField WHERE queryPage_id = ?))", pageId);
+    executeSql("DELETE FROM QueryQuestionAnswer WHERE queryField_id in (SELECT id from QueryField WHERE queryPage_id = ?)", pageId);
+    executeSql("DELETE FROM QueryOptionFieldOptionGroup WHERE optionField_id in (SELECT id from QueryField WHERE queryPage_id = ?)", pageId);
+    executeSql("DELETE FROM QueryOptionFieldOption WHERE optionField_id in (SELECT id from QueryField WHERE queryPage_id = ?)", pageId);
+    executeSql("DELETE FROM QueryOptionField WHERE id in (SELECT id from QueryField WHERE queryPage_id = ?)", pageId);
+    executeSql("DELETE FROM QueryTextField WHERE id in (SELECT id from QueryField WHERE queryPage_id = ?)", pageId);
+    executeSql("DELETE FROM QueryNumericField WHERE id in (SELECT id from QueryField WHERE queryPage_id = ?)", pageId);
+    executeSql("DELETE FROM QueryField WHERE queryPage_id = ?", pageId);
+    executeSql("DELETE FROM QueryPageSetting WHERE queryPage_id = ?", pageId);
+    executeSql("DELETE FROM QueryPage WHERE id = ?", pageId);
+  }
+
+  private void deleteQueryPageSetting(Long id) {
+    executeSql("delete from QueryPageSetting where id = ?", id);
+  }
+  
+  private List<Long> getPageIdsByType(String type) {
+    List<Long> result = new ArrayList<>();
+    
+    for (PageMeta page : pages) {
+      if (StringUtils.equals(page.getType(), type)) {
+        result.add(page.getId());
+      }
+    }
+    
+    return result;
+  }
+
+  private class PageMeta {
+
+    public PageMeta(long id, long queryId, long sectionId, String type) {
+      super();
+      this.id = id;
+      this.queryId = queryId;
+      this.sectionId = sectionId;
+      this.type = type;
+    }
+
+    private long id;
+    private long queryId;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1213)